Extra options to supply to gtkdoc-fixref for Solaris.

This change will not be sent upstream.

diff -ruN docs.orig/Makefile.am docs/Makefile.am
--- gtkspell-2.0.16/docs.orig/Makefile.am	2010-01-27 23:14:49.449868110 +0000
+++ gtkspell-2.0.16/docs/Makefile.am	2010-01-27 23:15:03.635550227 +0000
@@ -44,7 +44,7 @@
 
 # Extra options to supply to gtkdoc-fixref. Not normally needed.
 # e.g. FIXXREF_OPTIONS=--extra-dir=../gdk-pixbuf/html --extra-dir=../gdk/html
-FIXXREF_OPTIONS=
+FIXXREF_OPTIONS=--module=$(DOC_MODULE)
 
 # Used for dependencies. The docs will be rebuilt if any of these change.
 # e.g. HFILE_GLOB=$(top_srcdir)/gtk/*.h
